gitfind.html

```diff
@@ -9,14 +9,14 @@
             Github Compare
         </title>
     </head>
-    <body class="git">
+    <body class="git" style="background-image: url('./coffee.jpg');">

```

Comment:
check the image name from root directory.
Also, this feature is commented on line:13 in app.js

Reply:
> check the image name from root directory.
> Also, this feature is commented on line:13 in app.js

Yeah so whats the problem in that ? I wanted to use linear gradient in compare section as background and a proper background image in home section. Thats all

Reply:
Delete this now , i hope i have clear your doubts ✌🏼